### PR TITLE
Deploying 0.112.0 on mainnet-eu for V1 and V2.

### PR DESCRIPTION
**Pull request by mgoelswirlds**
**Description**:

Deploying 0.112.0 on mainnet-eu for V1 and V2.

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by steven-sheehy — DISMISSED** [review]

**Review by jascks — APPROVED** [review]
LGTM

File: clusters/mainnet-eu/mainnet-citus/helmrelease.yaml
Comment (jnels124) [inline clusters/mainnet-eu/mainnet-citus/helmrelease.yaml:1]: We may want to disable ingresses here temporarily to perform the cluster restart to take the stackgres changes. We will need to restart the cluster and the release status is not respected for this namespace. @steven-sheehy @xin-hedera

Comment (steven-sheehy) [reply clusters/mainnet-eu/mainnet-citus/helmrelease.yaml:1]: Sounds good

Comment (jnels124) [reply clusters/mainnet-eu/mainnet-citus/helmrelease.yaml:1]: if we go this route, disable acceptance as well and can be turned back on when we enable ingresses after cluster restart

**Review by steven-sheehy — APPROVED** [review]

**Review by jnels124 — APPROVED** [review]
LGTM

**Pull request merged**